app/code/Magento/Catalog/Ui/Component/Listing/Columns/ProductActions.php

```diff
@@ -58,7 +58,7 @@ public function prepareDataSource(array $dataSource)
                         'catalog/product/edit',
                         ['id' => $item['entity_id'], 'store' => $storeId]
                     ),
-                    'ariaLabel' => __('Edit ') . $item['name'],
+                    'ariaLabel' => __('Edit ') . (isset($item['name']) ? $item['name'] : ''),
```

Comment:
```suggestion
                    'ariaLabel' => __('Edit') . ' ' . ($item['name'] ?? ''),
```

- Thnx, wasn't aware nullish coalescing was a thing in PHP now
- The translation wasn't working anyway due to the added space at the end
- The parenthesis are absolutely NOT meaningless here, they enforce evaluation, without it, it still breaks for me (on PHP 7.4)
- I do absolutely agree that including it in the translation is best, but the issue at hand is possibly breaking admin views, and any other translations needs to be updated as well. I've checked if the pattern was accidentaly available but it is not

Reply:
@MeCapron Could you take a look at this, or tell me what I need to do to move this forward?